python/hsfs/core/online_store_sql_engine.py

```diff
@@ -557,16 +583,22 @@ async def _execute_prep_statements(
                 if key not in entries:
                     prepared_statements.pop(key)
 
-        tasks = [
-            asyncio.ensure_future(
-                self._query_async_sql(prepared_statements[key], entries[key])
-            )
-            for key in prepared_statements
-        ]
-        # Run the queries in parallel using asyncio.gather
-        results = await asyncio.gather(*tasks)
-        results_dict = {}
+        try:
```

Comment:
Will this lead to issues if running multiple feature views serving in the same runtime? You might need to use the FV name to distinguish them

Reply:
I tried to test with 2 different feature views in a loop and it worked. Each feature view should be called with `init_serving` first so that they get different connection pool objects